Gemfile.lock

```diff
@@ -30,12 +6,6 @@ GIT
       mongoid (>= 3.0, <= 4.0)
```

Comment:
This dependency results in mongoid being installed at exactly the version 4.0.0, instead of the bugfix release version 4.0.2.  I can't really imagine this is the actual intention.